src/zcbenchmarks.cpp

```diff
@@ -322,3 +326,82 @@ double benchmark_increment_note_witnesses(size_t nTxs)
     return timer_stop(tv_start);
 }
 
+// Fake the input of a given block
+class FakeCoinsViewDB : public CCoinsViewDB {
+    uint256 hash;
+    ZCIncrementalMerkleTree t;
+
+public:
+    FakeCoinsViewDB(std::string dbName, uint256& hash) : CCoinsViewDB(dbName, 100, false, false), hash(hash) {}
```

Comment:
Those parameters are already documented in `CCoinsViewDB`; setting them to false here is just copying the default parameters for a non-reindexing node (which has `fMemory = false` and `fWipe = fReindex`). The purpose of this class and constructor is solely to select a different database file, because the existing Bitcoin code assumes the only LevelDB used is the coins DB.